generator/ProtocolTestsGenerator/smithy-dotnet-protocol-test/smithy-build.json

```diff
@@ -0,0 +1,160 @@
+{
+  "version": "1.0",
+  "projections": {
+    "RestJsonProtocol": {
+      "transforms": [
+        {
+        "name": "includeServices",
+        "args": {
+          "services": ["aws.protocoltests.restjson#RestJson"]
+        }
+      },
+        {
+          "name": "excludeShapesBySelector",
```

Comment:
The smithy team provided some guidance on tests that we should leave out either due to issues with smithy2c2j or because the smithy2c2j artifact isn't up to date with the latest smithy release.

Reply:
I'm wondering about adding that context as comment here, so someone reviewing it later can know and also evaluate if it is time to reduce this list. Managing that in the smithy-build.json file probably isn't great. Could this be managed in ProtocolTestCustomization where you have logic to skip other things, and then you can have comment/context/TODOs there?

Reply:
I wanted to separate out the tests where entire operations are skipped and cases where we just skip invididual test id's. I thought if smithy provided these exclusion selectors I should use them. But, I can see how it would be hard to maintain why we skipped them later down the line. I'll move them.